README.md

```diff
@@ -0,0 +1,50 @@
+
+NOTE: This software is only if you want to *provide* the service.
+
+
+### INSTALLATION NOTES
+------------------
+
+- Pre-requisites
+
+	* Linux
+	* sendmail
+	* apache
+	* mysql
+	* perl
+
+- MySQL
+
+  ```
+	mysql_install_db
+	/etc/init.d/mysqld start
+	mysqladmin -uroot -p<pass>
+  ```
+
+	The following commands should be entered at the mysql prompt
+  ```
+	create database <sguser>
+	grant all privileges on <sguser>.* to <sguser> identified by '<sgpass>' with grant option
+	mysql -u<sguser> -p<sgpass> <db.sql
+	mysql -u<sguser> -p<sgpass> <dialogs.sql
+	```
+
+- Apache
+
+- Sendmail
+
+	* soft link from spameater to /etc/smrsh
+	* specify the soft link name in .forward
+
+- Spamgourmet
+
+	* make appropriate changes to spamgourmet.config
```

Comment:
Can you please move this down a little lower, just before the instruction to `make appropriate changes to ~/.forward`?